human_aware_rl/static/human_data/README.md

```diff
@@ -0,0 +1,97 @@
+# Human Experiment Data
+
+This subdirectory is the home of all human experiment data for the Overcooked game. All data was collected through Mturk and is fully anonymized. While the data collection code is proprietary, it relies heavily on the open source [Overcooked-demo](https://github.com/HumanCompatibleAI/overcooked-demo) project. 
```

Comment:
Data collection code is proprietary? As in our psiturk experiment repos? Those should be open source too?

Reply:
yeah the 'data collection code' refers to [overcooked-interactive-exp](https://github.com/micahcarroll/overcooked-interactive-exp) which is currently private. If you think it should be public we should probably add it to the HumanCompatibleAI github group, and then eventually make it part of `Overcooked-Demo`